scripts/bump_sui_testnet_version.sh

```diff
@@ -105,12 +105,15 @@ This PR updates the Sui testnet version to ${NEW_TAG}
 EOF
 )
 
+# Create PR
 PR_URL=$(gh pr create \
   --base main \
   --head "$BRANCH" \
   --title "chore: bump Sui version to ${NEW_TAG}" \
-  --reviewer "ebmifa,mlegner,wbbradley" \
+  --reviewer "MystenLabs/walrus-pe" \
   --body "$BODY" \
   2>&1 | grep -Eo 'https://github.com/[^ ]+')
 
+# Setting the PR to auto merge
+gh pr merge --auto --squash --delete-branch "$BRANCH"
```

Comment:
I don't fully understand the semantics here. Enabling "automerge" is fine, as long as all mandatory checks (approval, CI) are enforced.